starknet_py/tests/e2e/docs/code_examples/test_gateway_client.py

```diff
@@ -0,0 +1,207 @@
+# pylint: disable=unused-variable, redefined-builtin
```

Comment:
What's the builtin?

Reply:
`hash`

Reply:
Should we maybe rename it to something like `hash_` to avoid the redefinition?

Reply:
Maybe? Can do that